From 235ae1fec02f0f9cab85ee15e113e5d4770d4e19 Mon Sep 17 00:00:00 2001
From: Lukasz Rymanowski <lukasz.rymanowski@tieto.com>
Date: Fri, 22 Aug 2014 11:51:20 +0200
Subject: [PATCH 2/4] Enable 6LOWPAN

---
 defconfigs/bluetooth | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/defconfigs/bluetooth b/defconfigs/bluetooth
index 44e7bbd..d7b00b9 100644
--- a/defconfigs/bluetooth
+++ b/defconfigs/bluetooth
@@ -1,3 +1,5 @@
+CPTCFG_6LOWPAN=m
+CPTCFG_BT_6LOWPAN=m
 CPTCFG_BT=m
 CPTCFG_BT_RFCOMM=m
 CPTCFG_BT_BNEP=m
-- 
1.8.4

