From 911d2150d05f03fbcfec2ea8c4cf84b70b019dce Mon Sep 17 00:00:00 2001
From: Andrzej Kaczmarek <andrzej.kaczmarek@tieto.com>
Date: Fri, 14 Mar 2014 15:51:53 +0100
Subject: [PATCH] Add HCI H4

---
 defconfigs/bluetooth      | 2 ++
 drivers/bluetooth/Kconfig | 1 -
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/defconfigs/bluetooth b/defconfigs/bluetooth
index 44e7bbd..b370b50 100644
--- a/defconfigs/bluetooth
+++ b/defconfigs/bluetooth
@@ -3,3 +3,5 @@ CPTCFG_BT_RFCOMM=m
 CPTCFG_BT_BNEP=m
 CPTCFG_BT_BNEP_MC_FILTER=y
 CPTCFG_BT_BNEP_PROTO_FILTER=y
+CPTCFG_BT_HCIUART=m
+CPTCFG_BT_HCIUART_H4=y
diff --git a/drivers/bluetooth/Kconfig b/drivers/bluetooth/Kconfig
index 6056887..e11c420 100644
--- a/drivers/bluetooth/Kconfig
+++ b/drivers/bluetooth/Kconfig
@@ -29,7 +29,6 @@ config BT_HCIBTSDIO
 config BT_HCIUART
 	tristate "HCI UART driver"
 	depends on m
-	depends on TTY
 	help
 	  Bluetooth HCI UART driver.
 	  This driver is required if you want to use Bluetooth devices with
-- 
1.9.0

